index.d.ts

```diff
@@ -0,0 +1,4 @@
+export default class ImageCropElement extends HTMLElement {
+  src?: string
```

Comment:
https://github.com/github/markdown-toolbar-element/pull/13/files/c9ec083478b91bc8803514ea77eb40a9420a67ec#discussion_r308791257

```suggestion
  src: string || null
```